### ISSUE TITLE
Do UERANSIM support RAN network right now?

### ISSUE THREAD

**Issue by VLiu7** [aligungr/UERANSIM#606]
Hi @aligungr 

Does UERANSIM support RAN network slicing right now? 

In more detail, I would like to implement the following feature: a gNB sends different SIB1 to different UEs. If UERANSIM does not support it, can you give me some guidance on how to modify to support it?

Thank you very much!

**Comment by DiogoCruz40** [comment]
Yes it supports, check this [configuration](https://github.com/s5uishida/open5gs_5gc_ueransim_snssai_upf_sample_config).
